app/views/partners/show.html.erb

```diff
@@ -32,9 +32,78 @@
         <% else %>
           <%= render "show_header", partner: @partner, impact_metrics: @impact_metrics, partner_distributions: @partner_distributions %>
         <% end %>
-        <% if @partner.notes.present? %>
-          <%= render "notes", partner: @partner %>
-        <% end %>
+
+        <section class="card card-info card-outline">
+          <div class="card-header">
+            <h2 class="card-title">Contact Information</h2>
+          </div>
+          <div class="card-body p-3">
+            <div class="row px-2">
+              <div class="col-lg-4 col-sm-12">
+                <div>
+                  <h4> Executive Director </h4>
+                  <div>
+                    <label>Name:</label> <%= optional_data_text(@partner.profile.executive_director_name) %>
+                  </div>
+                  <div>
+                    <label>Email:</label> <%= optional_data_text(@partner.profile.executive_director_email) %>
+                  </div>
+                  <div>
+                    <label>Phone:</label> <%= optional_data_text(@partner.profile.executive_director_phone) %>
+                  </div>
```

Comment:
might be worth pulling these out into a shared partial although I see that program manager has a mobile while the others don't. Maybe we can make an issue for the future? I'm ok either way